### ISSUE TITLE
package golang.org/x/tools/go/gcimporter: unrecognized import path "golang.org/x/tools/go/gcimporter"

### ISSUE THREAD

**Issue by maylingge** [golang/lint#174]
When I try to install golint on ubuntu with "go get -u github.com/golang/lint/golint", I get the following errors:
package golang.org/x/tools/go/gcimporter: unrecognized import path "golang.org/x/tools/go/gcimporter"
package golang.org/x/tools/go/types: unrecognized import path "golang.org/x/tools/go/types"

I cannot find a way to resolve after searching among the issues and google.
Thank you.

**Comment by zimmski** [comment]
Is your GOPATH correct? Have a look at https://golang.org/doc/install#testing and https://github.com/golang/go/wiki/GOPATH and then do a

```bash
go get -u -v github.com/golang/lint/golint
```

**Comment by dsymonds** [comment]
You also may have an old x/tools version. Delete `$GOPATH/src/golang.org/x/tools` and start again.

**Comment by maylingge** [comment]
maylinge@ubuntu:~/go/src/github.com$ go get -u -v github.com/golang/lint/golint
github.com/golang/lint (download)
Fetching https://golang.org/x/tools/go/gcimporter?go-get=1
https fetch failed.
Still cannot get it. 
> echo $GOPATH
> /home/maylinge/go
Is it a network issue?

import "golang.org/x/tools/go/gcimporter": https fetch: Get https://golang.org/x/tools/go/gcimporter?go-get=1: dial tcp 74.125.68.141:443: i/o timeout
package golang.org/x/tools/go/gcimporter: unrecognized import path "golang.org/x/tools/go/gcimporter"
Fetching https://golang.org/x/tools/go/types?go-get=1
https fetch failed.
import "golang.org/x/tools/go/types": https fetch: Get https://golang.org/x/tools/go/types?go-get=1: dial tcp 74.125.68.141:443: i/o timeout
package golang.org/x/tools/go/types: unrecognized import path "golang.org/x/tools/go/types"

Thank you.

**Comment by zimmski** [comment]
Yep, that timeout looks like a network issue. This is not a problem with golint. Please start a thread on https://groups.google.com/forum/#!forum/golang-nuts

**Comment by maylingge** [comment]
Thank you.

2015-11-19 0:00 GMT+08:00 Markus Zimmermann <notifications@github.com>:
